src/main/resources/create-script.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TABLE `bank_service`.`new_table` (
```

Comment:
what is the table name, please use the appropriate table name.

Reply:
What happens if there is already a table with same name? Will that throw error? please handle that.

Reply:
Also, add indexes and unique key as well.